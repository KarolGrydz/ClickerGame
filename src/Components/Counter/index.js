import React from 'react';

const Counter = props => <span>{props.value || 0}</span>;

export default Counter;
