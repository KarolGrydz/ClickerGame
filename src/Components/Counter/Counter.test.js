import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Counter from './index';

describe('<Counter />', () => {
  it('should be defined', () => {
    expect(Counter).toBeDefined();
  });
  it('should render without crashing', () => {
    const wrapper = shallow(<Counter />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
  it('props should be numer', () => {
    const wrapper = shallow(<Counter />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
