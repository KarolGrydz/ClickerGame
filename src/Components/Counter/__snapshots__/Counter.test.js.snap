// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Counter /> props should be numer 1`] = `
<span>
  0
</span>
`;

exports[`<Counter /> should render after passing a number 1`] = `
<span>
  0
</span>
`;

exports[`<Counter /> should render without crashing 1`] = `
<span>
  0
</span>
`;
